Adafruit_LSM6DS.cpp

```diff
@@ -142,19 +133,19 @@ boolean Adafruit_LSM6DS::begin_I2C(uint8_t i2c_address, TwoWire *wire,
  *    @brief  Sets up the hardware and initializes hardware SPI
  *    @param  cs_pin The arduino pin # connected to chip select
  *    @param  theSPI The SPI object to be used for SPI connections.
+ *    @param  frequency The SPI bus frequency
  *    @param  sensor_id
  *            The user-defined ID to differentiate different sensors
  *    @return True if initialization was successful, otherwise false.
  */
 bool Adafruit_LSM6DS::begin_SPI(uint8_t cs_pin, SPIClass *theSPI,
-                                int32_t sensor_id) {
+                                uint32_t frequency, int32_t sensor_id) {
```

Comment:
please add freq after sensorid so we dont break existing examples!

Reply:
I thought about that too and came to the conclusion, that it is more often you define the frequency than you define the sensor ID. But I guess, not breaking old code is more important :grinning: